src/components/ShoppingList/styles.module.css

```diff
@@ -34,7 +34,7 @@
 
 .newItem input {
     border-bottom: 2px solid var(--main_green);
-    font-size: 12px;
+    font-size: 16px;
```

Comment:
This is the minimum size to stop the zooming on safari, it looks a bit odd as it's now bigger than the rest of the text but we should probably increase the size of the other text too.